knowledge-base/azure/k8s-deploy/action-security.yml

```diff
@@ -0,0 +1,14 @@
+name: 'Deploy to Kubernetes cluster'
+github-token:
+  action-input:
+    input: token
+    is-default: true
+  permissions:
+    contents: read
```

Comment:
I did not see contents: read or issues: write.
This just needs `actions: read` and the link is https://github.com/Azure/k8s-deploy/blob/ac49626466364585521685ccaad17e499a8263a8/src/githubClient.ts#L11

Reply:
@arjundashrath this one is not updated.